app/livechat/client/views/app/tabbar/visitorEdit.html

```diff
@@ -30,6 +30,17 @@ <h3>{{username}}</h3>
 						</div>
 					</label>
 				</div>
+
+				{{#each visitorCustomFields}}
+					<div class="rc-input rc-form-group rc-form-group--small">
+						<label class="rc-input__label">
+							<div class="rc-input__title">{{label}}</div>
+							<div class="rc-input__wrapper">
+								<input class="rc-input__element" type="text" name="{{name}}" autocomplete="off" data-visitorLivechatData="true" value="{{value}}">
```

Comment:
Let's create a new permission to allow/deny editing `custom field` values, what do you think?
By default, The new permission would be assigned to the `livechat-agent`, `livechat-managers` and `admin`.

What do you think?

Reply:
Do I need to add these permissions for the REST API?

Reply:
> Do I need to add these permissions for the REST API?

No, It's not necessary.
Since we only allow fetching data through the REST API, we don't need to change the REST API permissions.

Reply:
I meant the endpoint for updating values POST `livechat/custom.field` (and `livechat/custom.fields`). If we restrict access to editing values of `custom fields` for GUI, the user can still change the values through these API methods.
I think that if we restrict the permissions to edit values of `custom fields` in the interface, then these restrictions should be for the API. What do you think?

Reply:
> I meant the endpoint for updating values POST `livechat/custom.field` (and `livechat/custom.fields`). If we restrict access to editing values of `custom fields` for GUI, the user can still change the values through these API methods.
> I think that if we restrict the permissions to edit values of `custom fields` in the interface, then these restrictions should be for the API. What do you think?

I see.. The fact is that the `Livechat Widget` calls those `endpoits` too, so we can't add permission restrictions there because there is no authenticated user on the `Widget-side`, you know?
So, let's just skip the idea of implementing new permissions to deal with `custom fields`, okay?

Reply:
I did not know... Thank you. Ok, then I won't add new permissions

Reply:
Perfect!
If you're gonna add the validation option, I'll wait for your new commit to get back to the review process, okay?

Reply:
Yes, I will add and let you know